app/app.coffee

```diff
@@ -16,6 +17,7 @@ else
   config.product = ENUMS.PRODUCT.DEVKNOX
 
 installIntercom()
+surveyMonkey()
```

Comment:
Man, dont install survey monjey here - link we discussed, keep a button for feedback.